src/cms/api/redirects/models/redirects.settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "kind": "collectionType",
+  "collectionName": "redirects",
```

Comment:
`redirects`  shouldn't be plural.  All the other collection types names are singular.

We might even want to call it something like `legacy-redirect` instead, since **redirect** is almost like a reserved word in web development.